### PR TITLE
Fixing KeyManager DB script issues

### PR DESCRIPTION
**Pull request by CrowleyRajapakse**
Fixing https://github.com/wso2/product-apim/issues/8600

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql [hunk 1]
```diff
@@ -1832,7 +1832,7 @@ CREATE TABLE IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql:1835]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql [hunk 2]
```diff
@@ -1844,7 +1844,7 @@ CREATE TABLE IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql:1847]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql [hunk 3]
```diff
@@ -1873,7 +1873,7 @@ CREATE TABLE IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql:1876]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql_cluster.sql [hunk 4]
```diff
@@ -1957,7 +1957,7 @@ CREATE TABLE IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BLOB NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql_cluster.sql:1960]: This should not have a NULL default value. Please pick appropriate default value 0 or 1.

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/postgresql.sql [hunk 5]
```diff
@@ -2127,7 +2127,7 @@ CREATE TABLE  IF NOT EXISTS AM_KEY_MANAGER (
   DESCRIPTION VARCHAR(256) NULL,
   TYPE VARCHAR(45) NULL,
   CONFIGURATION BYTEA NULL,
-  ENABLED INT NULL,
+  ENABLED BOOLEAN NULL,
```
Comment (uvindra) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/postgresql.sql:2130]: This should be ENABLED BOOLEAN DEFAULT '1'. Please pick appropriate default value 0 or 1.

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql (hunk 1 above)
Comment (CrowleyRajapakse) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql]: resolved

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql (hunk 2 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/h2.sql]: resolved

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql (hunk 3 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql.sql]: resolved

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql_cluster.sql (hunk 4 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mysql_cluster.sql]: resolved

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/postgresql.sql (hunk 5 above)
Comment (CrowleyRajapakse) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/postgresql.sql]: resolved

**Pull request merged**